src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Multiline.Edit.cshtml

```diff
@@ -1,6 +1,7 @@
 @model OrchardCore.ContentFields.ViewModels.EditHtmlFieldViewModel
-@using OrchardCore.ContentManagement.Metadata.Models
 @using OrchardCore.ContentFields.Settings
+@using OrchardCore.ContentManagement.Metadata.Models
```

Comment:
Can we move the namespaces to `_ViewImports.cshtml`, while there are referenced a lot in many views?